syntax = "proto3";

package config.notification;

service TemplateMailController {
    rpc List(TemplateMailListRequest) returns (TemplateMailListResponse) {}
    rpc Retrieve(TemplateMailRetrieveRequest) returns (TemplateMailResponse) {}
}

message TemplateMailListRequest {
}

message TemplateMailListResponse {
    repeated TemplateMailResponse results = 1;
}

message TemplateMailResponse {
    string id = 1;
    string content_type = 2;
    string subject = 3;
    string body_html = 4;
    string body_text = 5;
}

message TemplateMailRetrieveRequest {
    string content_type = 1;
}

